spec/index.bs

```diff
@@ -5213,17 +5213,13 @@ dictionary GPUExternalTextureDescriptor
         const videoElement = document.createElement('video');
         // ... set up videoElement, wait for it to be ready...
 
-        let externalTexture;
-
         function frame() {
             requestAnimationFrame(frame);
 
-            // Re-import only if necessary
-            if (!externalTexture || externalTexture.expired) {
-                externalTexture = gpuDevice.importExternalTexture({
-                    source: videoElement
-                });
-            }
+            // Always re-import, because textures are cached internally.
```

Comment:
Examples are one of the first places developers will look, so let's fully document the behavior right here.

```suggestion
            // Always re-import the video on every animation frame, because the
            // import is likely to have expired.
            // The browser may cache and reuse a past frame, and if it does it
            // may return the same GPUExternalTexture object again.
            // In this case, old bind groups are still valid.
```